### PR TITLE
blog style readme for torch sparse-quant TL notebook

### PR DESCRIPTION
**Pull request by bfineran**
the purpose of this document is to provide a blog-like landing page that gives a quick primer and motivation for sparse-quantized transfer learning and how to quickly get started using it with SparseML

### CODE REVIEW

**Comment by bfineran** [comment]
I'm very open to what goes in here. @reviewers please feel free to make suggestions

File: examples/pytorch_sparse_quantized_transfer_learning/README.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+software distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# PyTorch Sparse-Quantized Transfer Learning with SparseML
+
+[Pruning](https://neuralmagic.com/blog/pruning-overview/) and
+[quantization](https://arxiv.org/abs/1609.07061) are well established methods for accelerating
```
Comment (jeanniefinks) [inline examples/pytorch_sparse_quantized_transfer_learning/README.md:20]:
```suggestion
[quantization](https://arxiv.org/abs/1609.07061) are well-established methods for accelerating
```

File: examples/pytorch_sparse_quantized_transfer_learning/README.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+neural networks.  Individually, both methods yield significant speedups for CPU inference
+(a theoretical maximum of 4x for INT8 quantization) and can make CPU deployments an attractive
+option for real time model inference.
```
Comment (jeanniefinks) [inline examples/pytorch_sparse_quantized_transfer_learning/README.md:23]:
```suggestion
option for real-time model inference.
```

File: examples/pytorch_sparse_quantized_transfer_learning/README.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+Sparse-quantized models leverage both techniques and can achieve speedups upwards of 6-7x when using
+the [DeepSparse Engine](https://github.com/neuralmagic/deepsparse) with
+[compatible hardware](https://docs.neuralmagic.com/deepsparse/hardware.html).
+
+Using powerful [SparseML](https://github.com/neuralmagic/sparseml) recipes, it is easy to create sparse-quantized models.
+Additionally, the SparseML team is actively creating pre-trained sparse-quantized models that maintain accuracy
+targets and achieve high CPU speedups - and it is easy to leverage these models for speedups with your own datasets
+using sparse-quantized transfer learning.
+
+Sparse-quantized transfer learning takes place in two phases:
```
Comment (jeanniefinks) [inline examples/pytorch_sparse_quantized_transfer_learning/README.md:34]:
```suggestion
Transfer learning is an ML approach where a model developed for solving for one task is reused and applied to another related but different task. Sparse-quantized transfer learning takes place in two phases:
```

File: examples/pytorch_sparse_quantized_transfer_learning/README.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+## ResNet-50 Imagenette Example
+
+The [SparseZoo](https://github.com/neuralmagic/sparseml) hosts a sparse-quantized ResNet-50 model trained
+on the ImageNet dataset.  It maintains 99% of the baseline accuracy and can achieve over 6.5x
+speedup using the DeepSparse Engine.  There are multiple paths to explore sparse-quantized
+transfer learning with this model.
+
+### Notebook
+`sparseml/examples/pytorch_sparse_quantized_transfer_learning/pytorch_sparse_quantized_transfer_learning.ipynb`
+is a jupyter notebook that provides a step-by-step walk-through for
```
Comment (jeanniefinks) [inline examples/pytorch_sparse_quantized_transfer_learning/README.md:52]:
```suggestion
is a Jupyter Notebook that provides a step-by-step walk-through for
```

**Review by jeanniefinks — DISMISSED** [review]
Great refinement on transfer learning def!

File: examples/pytorch_sparse_quantized_transfer_learning/README.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+
+# PyTorch Sparse-Quantized Transfer Learning with SparseML
+
+[Pruning](https://neuralmagic.com/blog/pruning-overview/) and
+[quantization](https://arxiv.org/abs/1609.07061) are well-established methods for accelerating
+neural networks.  Individually, both methods yield significant speedups for CPU inference
+(a theoretical maximum of 4x for INT8 quantization) and can make CPU deployments an attractive
+option for real-time model inference.
+
+Sparse-quantized models leverage both techniques and can achieve speedups upwards of 6-7x when using
```
Comment (markurtz) [inline examples/pytorch_sparse_quantized_transfer_learning/README.md:25]: can we get a link out for the 6-7x to the resnet50 blog? Also, can we change to just 7x or ~7x?

File: examples/pytorch_sparse_quantized_transfer_learning/README.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# PyTorch Sparse-Quantized Transfer Learning with SparseML
+
```
Comment (markurtz) [inline examples/pytorch_sparse_quantized_transfer_learning/README.md:18]: grounding this with a product flow image would work well I think as discussed in slack, adding the file to this comment for the diagram as discussed

[flow-sparsification-model_sparsezoo-recipe_sparsezoo-sparsifcation_sparseml_training-deployment_deepsparse.svg.zip](https://github.com/neuralmagic/sparseml/files/6043774/flow-sparsification-model_sparsezoo-recipe_sparsezoo-sparsifcation_sparseml_training-deployment_deepsparse.svg.zip)

**Review by markurtz — CHANGES_REQUESTED** [review]

**Review by jeanniefinks — DISMISSED** [review]

**Review by markurtz — APPROVED** [review]

**Review by mgoin — APPROVED** [review]

**Pull request merged**